docs/themes/hugo-geekdoc/layouts/partials/pagination.html

```diff
@@ -1,19 +1,19 @@
-{{ $pag := $.Paginator }}
+{{ $page := $.Paginator }}
 
 
 <nav class="gdoc-paging flex flex-even align-center" role="navigation">
   <div class="gdoc-paging__item gdoc-paging__item--prev">
-    {{ if $pag.HasPrev }}
-      <a class="flex-inline align-center fake-link no-wrap" href="{{ $pag.Prev.URL }}">
+    {{ if $page.HasPrev }}
+      <a class="flex-inline align-center fake-link no-wrap" href="{{ $page.Prev.URL }}">
         <i class="gdoc-icon">gdoc_keyboard_arrow_left</i>
         {{ i18n "pagination_page_prev" | upper }}
       </a>
     {{ end }}
   </div>
-  <div class="gdoc-paging__state">{{ i18n "pagination_page_state" $pag }}</div>
+  <div class="gdoc-paging__state">{{ i18n "pagination_page_state" $page }}</div>
   <div class="gdoc-paging__item gdoc-paging__item--next">
-    {{ if $pag.HasNext }}
-      <a class="flex-inline align-center fake-link no-wrap" href="{{ $pag.Next.URL }}">
+    {{ if $page.HasNext }}
```

Comment:
This is from the Hugo theme, which we should not change directly. Do you know if updating the theme will resolve the issue?